src/wp-includes/media.php

```diff
@@ -5449,3 +5449,283 @@ function wp_increase_content_media_count( $amount = 1 ) {
 
 	return $content_media_count;
 }
+
+/**
+ * Add the dominant color metadata to the attachment.
+ *
+ * @since 1.2.0
+ *
+ * @param array $metadata      The attachment metadata.
+ * @param int   $attachment_id The attachment ID.
+ * @return array $metadata The attachment metadata.
+ */
+function dominant_color_metadata( $metadata, $attachment_id ) {
```

Comment:
We can just add this directly there 
https://github.com/WordPress/wordpress-develop/blob/a14324ca3af4eeff69406d3e59719a33f055f07e/src/wp-admin/includes/image.php#L646

Reply:
filter removed and calling function directly